electrumsv/restapi.py

```diff
@@ -0,0 +1,227 @@
+import asyncio
+import json
+from base64 import b64decode
+from typing import Optional, Dict, Union, Any
+from aiohttp import web
+from .logs import logs
+from .app_state import app_state
+from .util import to_bytes, to_string, constant_time_compare
+
+
+def get_app_state():
```

Comment:
I'm not that bothered about this, but I suspect there's a better way to do this with a fixture or something in the pytest-based tests.

Reply:
I did try but with all of the metaclass shenanigans... I just wasn't able to find a way to get it working in a timely manner. I don't think it's worth the time and effort.